web/gui/dashboard_info.js

```diff
@@ -1317,20 +1363,20 @@ netdataDashboard.context = {
         info: 'Calls to the functions responsible for closing (<a href="https://www.informit.com/articles/article.aspx?p=370047&seqNum=4" target="_blank">do_exit</a>) and releasing (<a  href="https://www.informit.com/articles/article.aspx?p=370047&seqNum=4" target="_blank">release_task</a>) tasks.'
     },
 
-    'apps.bandwidth_sent': {
+    'apps.total_bandwidth_sent': {
         info: 'Bytes sent by functions <code>tcp_sendmsg</code> and <code>udp_sendmsg</code>.'
     },
 
-    'apps.bandwidth_recv': {
+    'apps.total_bandwidth_recv': {
         info: 'Bytes received by functions <code>tcp_cleanup_rbuf</code> and <code>udp_recvmsg</code>. We use <code>tcp_cleanup_rbuf</code> instead <code>tcp_recvmsg</code>, because this last misses <code>tcp_read_sock()</code> traffic and we would also need to have more probes to get the socket and package size.'
     },
 
     'apps.bandwidth_tcp_send': {
-        info: 'Calls for function <code>tcp_sendmsg</code>.'
+        info: 'Calls to function <code>tcp_sendmsg</code>.'
     },
 
     'apps.bandwidth_tcp_recv': {
-        info: 'Calls for functions <code>tcp_cleanup_rbuf</code>. We use <code>tcp_cleanup_rbuf</code> instead <code>tcp_recvmsg</code>, because this last misses <code>tcp_read_sock()</code> traffic and we would also need to have more probes to get the socket and package size.'
+        info: 'Calls to functions <code>tcp_cleanup_rbuf</code>.'
```

Comment:
That is the copy of the chart title.

<img width="381" alt="Screenshot 2021-08-19 at 18 28 53" src="https://user-images.githubusercontent.com/22274335/130096974-a1c45bca-a2a7-4530-a828-f495120c6fae.png">

Why do we need that? Also, "tcp_cleanup_rbuf" doesn't say much, it is hard (assuming not a specialist) to get that we use it as "received" => that could be in the "info".

---

> That is the copy of the chart title.

That is not the only chart with that _problem_.

Reply:
I pushed a commit where I am updating the dashboard_info text.